src/components/App/ExpressRecorder.tsx

```diff
@@ -757,33 +797,33 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                         </button>
                     )}
                     {showUploadUI &&
-                        !doRecording &&
-                        recordedBlobs.length > 0 &&
-                        !this.uploadedOnce && (
-                            <div className={`${styles["express-recorder__bottom"]}`}>
-                                <button
-                                    className={`xr_btn xr_btn__download ${styles["bottom__btn"]} ${styles["btn__clear"]} ${styles["btn__download"]} `}
-                                    onClick={this.saveFile}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Download a Copy")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn__reset ${styles["bottom__btn"]} ${styles["btn__clear"]}`}
-                                    onClick={this.recordAgain}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Record Again")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn-primary xr_btn__save ${styles["bottom__btn"]} ${styles["btn__save"]}`}
-                                    onClick={this.initiateUpload}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Use This")}
-                                </button>
-                            </div>
-                        )}
+                    !doRecording &&
+                    (recordedBlobs.length || screenRecordedBlobs.length) &&
+                    !this.uploadedOnce ? (
```

Comment:
move this condition into a variable that will hold your answer.
please add on line **689**:
```
const showControllers = showUploadUI &&
                    !doRecording &&
                    (recordedBlobs.length || screenRecordedBlobs.length) &&
                    !this.uploadedOnce;
```
and use`showControllers` variable for your conditioin.

Reply:
done